lib/modules/manager/unity3d/index.ts

```diff
@@ -0,0 +1,11 @@
+import type { Category } from '../../../constants';
+import { Unity3dDatasource } from '../../datasource/unity3d';
+export { extractPackageFile } from './extract';
+
+export const defaultConfig = {
+  managerFilePatterns: ['/(^|/)ProjectSettings/ProjectVersion\\.txt$/'],
```

Comment:
```suggestion
  managerFilePatterns: ['**/ProjectSettings/ProjectVersion.txt'],
```

glob matching here is simpler